src/sonic-yang-models/yang-models/sonic-configurable-drop-monitor.yang

```diff
@@ -0,0 +1,51 @@
+module sonic-configurable-drop-monitor {
+    yang-version 1.1;
+
+    namespace "http://github.com/sonic-net/sonic-configurable-drop-monitor";
+
+    prefix configurable-drop-monitor;
+
+    description "DEBUG_DROP_MONITOR YANG Module for SONiC OS";
+
+    revision 2025-01-10 {
+        description
+            "First revision";
+    }
+
+    container sonic-configurable-drop-monitor {
+        container DEBUG_DROP_MONITOR {
```

Comment:
Hi @theasianpianist , please check if `DEBUG_COUNTER` and `DEBUG_COUNTER_DROP_REASON`has dependency on this for github issue missing YANG: https://github.com/sonic-net/sonic-buildimage/issues/21505